.etl.env

```diff
@@ -0,0 +1,27 @@
+ETL_CLICK_API=https://clickrevaha-sys.molsa.gov.il/api/solr?rows=1000
+
+ETL_GUIDESTAR_USERNAME=
+ETL_GUIDESTAR_PASSWORD=
+ETL_GUIDESTAR_API=https://www.guidestar.org.il/services/apexrest/api
+
+ETL_GOVMAP_API_KEY=
+ETL_GOVMAP_AUTH=https://ags.govmap.gov.il/Api/Controllers/GovmapApi/Auth
+ETL_GOVMAP_REQUEST_ORIGIN=https://www.kolzchut.org.il
+ETL_GOVMAP_GEOCODE_API=https://ags.govmap.gov.il/Api/Controllers/GovmapApi/Geocode
+
+ETL_AIRTABLE_BASE=appF3FyNsyk4zObNa
+ETL_AIRTABLE_VIEW='Grid view'
+ETL_AIRTABLE_LOCATION_TABLE=Locations
+ETL_AIRTABLE_ORGANIZATION_TABLE=Organizations
+ETL_AIRTABLE_BRANCH_TABLE=Branches
+ETL_AIRTABLE_SITUATION_TABLE=Situations
+
+ETL_MAPBOX_ACCESS_TOKEN=
+ETL_MAPBOX_LIST_TILESETS=https://api.mapbox.com/tilesets/v1/srm-kolzchut
+ETL_MAPBOX_UPLOAD_CREDENTIALS=https://api.mapbox.com/uploads/v1/srm-kolzchut/credentials
+ETL_MAPBOX_CREATE_UPLOAD=https://api.mapbox.com/uploads/v1/srm-kolzchut
+ETL_MAPBOX_UPLOAD_STATUS=https://api.mapbox.com/uploads/v1/srm-kolzchut/
+
+ETL_DATAFLOWS_AIRTABLE_APIKEY=
+
+GOOGLE_API_KEY=
```

Comment:
See comment on other PR - we should probably rename this to `ETL_GOOGLE_MAPS_API_KEY` (while fixing the code accordingly) for consistency